File: Groupe1/TP1/src/cercle.c
// #include <stdio.h>

// #define PI 3.14159

// int main() {
//     float rayon, aire, perimetre;

//     printf("Entrez le rayon du cercle : ");
//     scanf("%f", &rayon);

//     aire = PI * rayon * rayon;
//     perimetre = 2 * PI * rayon;

//     printf("L'aire du cercle est : %.2f\n", aire);
//     printf("Le périmètre du cercle est : %.2f\n", perimetre);

//     return 0;
// }

#include <stdio.h>

int main() {

    float aire, perimetre, rayon, pi;

    pi = 3.14159;

    printf("Entrez le rayon du cercle: ");
    scanf("%f", &rayon);

    aire = pi*rayon*rayon;

    perimetre = 2*pi*rayon;

    printf("l'aire du cercle est : %f\n", aire);

    printf("le perimetre du cercle est : %f\n", perimetre);

    return 0;
}

File: Groupe1/TP1/src/cercle.py
import math

rayon = float(input("Entrez le rayon du cercle : "))

aire = math.pi * rayon * rayon
perimetre = 2 * math.pi * rayon

print(f"L'aire du cercle est : {aire:.2f}")   #pour n'avoir que 2 chiffres après la virgule
print("Le périmètre du cercle est : ", perimetre)
